doc/ipython-notebooks/clustering/GMM.ipynb

```diff
@@ -11,7 +11,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### By Heiko Strathmann - <a href=\"mailto:heiko.strathmann@gmail.com\">heiko.strathmann@gmail.com</a> - <a href=\"github.com/karlnapf\">github.com/karlnapf</a> - <a href=\"herrstrathmann.de\">herrstrathmann.de</a>.   Based on the GMM framework of the <a href=\"https://www.google-melange.com/gsoc/project/google/gsoc2011/alesis_novik/11001\">Google summer of code 2011 project</a> of Alesis Novik - <a href=\"https://github.com/alesis\">https://github.com/alesis</a>"
+    "#### By Heiko Strathmann - [heiko.strathmann@gmail.com](mailto:heiko.strathmann@gmail.com) - [github.com/karlnapf](http://github.com/karlnapf) - [herrstrathmann.de](http://herrstrathmann.de).\n",
```

Comment:
It's acting strangely...
It works if I open it in new tab. Also working properly when opened using jupyter.
Only gives error when clicked normally in github preview.
Suggestions?